delfin/db/sqlalchemy/models.py

```diff
@@ -286,3 +287,115 @@ class FailedTask(BASE, DelfinBase):
     job_id = Column(String(36))
     deleted_at = Column(DateTime)
     deleted = Column(Boolean, default=False)
+
+
+class StorageInitiator(BASE, DelfinBase):
```

Comment:
Please remove `Storage` prefix for models if it not adds clarity

Reply:
Updated StorageInitiator